### PR TITLE
Laravel 8 support

### LINKED ISSUE #42: Laravel 8 Support
Laravel 8 has been released today and we need to update this repo to support it.
See https://laravel.com/docs/8.x/upgrade & https://github.com/laravel-notification-channels/channels/issues/107

### PR DESCRIPTION
**Pull request by Cannonb4ll**
Closes #42

### CODE REVIEW

**Comment by casperboone** [comment]
Hi @Cannonb4ll, thx for the PR! Did you test the package in a Laravel 8 app?

File: composer.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     "require": {
         "php": ">=7.2",
         "guzzlehttp/guzzle": "^6.2",
-        "illuminate/notifications": "~5.5 || ^6.0 || ^7.0",
-        "illuminate/support": "~5.5 || ^6.0 || ^7.0"
+        "illuminate/notifications": "~5.5 || ^6.0 || ^7.0 || ^8.0",
```
Comment (atymic) [inline composer.json:17]: Maybe worth dropping 5.x since it's fully out of support now.

**Comment by Cannonb4ll** [comment]
@casperboone Yes, worked on a L8.0.2 app!

File: composer.json (hunk 1 above)
Comment (Cannonb4ll) [reply composer.json:17]: I'll let @casperboone decide on that, if he agree's I can remove it.

Comment (casperboone) [reply composer.json:17]: Let's drop that once we do a somewhat larger general refresh of the package :)

**Comment by casperboone** [comment]
Thx! 😄

**Pull request merged**